src/main/java/org/apache/bcel/generic/MethodGen.java

```diff
@@ -237,6 +238,22 @@ public MethodGen(final Method m, final String class_name, final ConstantPoolGen
                             addLocalVariable(l.getName(), Type.getType(l.getSignature()), l
                                     .getIndex(), start, end);
                         }
+                    } else if (a instanceof LocalVariableTypeTable) { // generic variables by sam.
```

Comment:
We don't add author tags to our code. We add contirbutors to the `<contributors>` section of pom.xml.